templates/layout/page--ecl-branding--standardised.html.twig

```diff
@@ -19,7 +19,21 @@
 {% block navigation %}
   <div class="ecl-site-header-standardised__banner">
     <div class="ecl-container">
-      <div class="ecl-site-header-standardised__site-name">{{ site_name }}</div>
+      {% if site_name is not empty %}
```

Comment:
It would be nice to have also test coverage. I think it would be possible to do this in `The demo site header features placeholder blocks` scenario.